requirements.txt

```
@@ -40,6 +40,7 @@ jedi==0.14.1
 Jinja2==2.10.1
 jmespath==0.9.4
 joblib==0.13.2
+joist>=0.0.2
```

For now let's pin it directly.
```suggestion
joist==0.0.2
```